### ISSUE TITLE
No module named core

### ISSUE THREAD

**Issue by TheLelex** [daducci/COMMIT#29]
I am trying to install Commit on a Debian 9 machine with anaconda python. I followed the installation instruction (AMICO python code ) and then just did pip install . on COMMIT directory. I have no error but when I try to import commit from my python shell I have 
 ImportError : No module named core

If you have any idea of what I did wrong or a hint for the error. I precise that import amico worked perfectly.

**Comment by daducci** [comment]
Hi @TheLelex ,

what was the issue?

**Comment by TheLelex** [comment]
Hello Mister Daducci,

it was not a real issue sorry for that. I installed Amico and Commit to
test it on HCP dataset. Then  tried to import commit module from the same
directory (Commit directory) making a circular import
so that I got the error No module named core. Actually everything works
just fine. Sorry about that.

In addition, I may have some questions regarding COMMIT results with
respect to tractography algorithms used (already developped in your paper)
what would be the most convenient mail address to contact you ?

Best regards,


2017-10-27 12:18 GMT+02:00 Alessandro Daducci <notifications@github.com>:
